pkgs/os-specific/linux/zfs/generic.nix

```diff
@@ -137,6 +155,14 @@ let
               "bashcompletiondir=$out/share/bash-completion/completions"
 
           substituteInPlace ./cmd/arc_summary --replace-fail "/sbin/modinfo" "modinfo"
+        ''
+        + ''
+          echo 'Supported Kernel versions:'
+          grep '^Linux-' META
+          echo 'Checking kernelMinSupportedMajorMinor is correct...'
+          grep --quiet '^Linux-Minimum: *${lib.escapeRegex kernelMinSupportedMajorMinor}$' META
+          echo 'Checking kernelMaxSupportedMajorMinor is correct...'
+          grep --quiet '^Linux-Maximum: *${lib.escapeRegex kernelMaxSupportedMajorMinor}$' META
```

Comment:
Is it fatal if this fails? If so, how does this interact with kernelMaxBuildableMajorMinor?